jerry-libc/include/string.h

```diff
@@ -19,18 +19,19 @@
 #include <stddef.h>
 
 #ifdef __cplusplus
-# define EXTERN_C "C"
-#else /* !__cplusplus */
-# define EXTERN_C
+extern "C" {
 #endif /* !__cplusplus */
 
-extern EXTERN_C int    memcmp (const void *s1, const void *s2, size_t n);
-extern EXTERN_C void*  memcpy (void *dest, const void *src, size_t n);
-extern EXTERN_C void*  memset (void *s, int c, size_t n);
-extern EXTERN_C int    strcmp (const char *s1, const char *s2);
-extern EXTERN_C size_t strlen (const char *s);
-extern EXTERN_C void*  memmove (void *dest, const void *src, size_t n);
-extern EXTERN_C int    strncmp (const char *s1, const char *s2, size_t n);
-extern EXTERN_C char*  strncpy (char *dest, const char *src, size_t n);
+int    memcmp (const void *s1, const void *s2, size_t n);
+void*  memcpy (void *dest, const void *src, size_t n);
+void*  memset (void *s, int c, size_t n);
+int    strcmp (const char *s1, const char *s2);
+size_t strlen (const char *s);
+void*  memmove (void *dest, const void *src, size_t n);
+int    strncmp (const char *s1, const char *s2, size_t n);
+char*  strncpy (char *dest, const char *src, size_t n);
```

Comment:
If we are alread touching style issues: I'd drop the alignment spaces here. Nowhere else do we use them..